Marlin/src/lcd/ultralcd.cpp

```diff
@@ -1311,7 +1316,9 @@ void kill_screen(const char* lcd_msg) {
     #if ENABLED(MESH_BED_LEVELING) && ENABLED(LCD_BED_LEVELING)
       MENU_ITEM_EDIT(float43, MSG_BED_Z, &mbl.z_offset, -1, 1);
     #endif
-
+    #if ENABLED(ENABLE_LEVELING_FADE_HEIGHT)
+      MENU_MULTIPLIER_ITEM_EDIT_CALLBACK(float3, MSG_Z_FADE_HEIGHT, &new_z_fade_height, 0.0, 100.0, _lcd_set_z_fade_height);
+    #endif
```

Comment:
Error: The Tune sub-menu is only available during active movement. It may be bad to modify `planner.z_fade_height` during printing with leveling compensation enabled because it modifies `current_position[Z_AXIS]`. Presumably the new value of `current_position[Z_AXIS]` corresponds to the kinematic `stepper.count_position[...]` values, but due to the natural imprecision of `float` numbers, this may cause the nozzle to move up and down by some amount when editing is done.